src/index.html

```diff
@@ -22,6 +22,346 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div class="container">
+      <header class="head head--flex">
+        <div class="head__item1">
+          <a
+            class="head-logo-link"
+            href="#"
+          >
+            <img
+              src="images/logo.png"
+              alt="picture logo"
+            />
+          </a>
+        </div>
+        <div class="head__item2">
+          <nav class="head-nav">
+            <ul class="head-nav__list">
+              <li>
+                <a
+                  class="is-active"
+                  href="#"
+                >
+                  Apple
+                </a>
+              </li>
+              <li>
+                <a href="#">Samsung</a>
+              </li>
+              <li>
+                <a href="#">Smartphones</a>
+              </li>
+              <li>
+                <a
+                  href="#"
+                  data-qa="nav-hover"
```

Comment:
The `data-qa` attribute is set to `nav-hover` here, which is inconsistent with other uses of `hover` or `card-hover`. Ensure consistency across all elements.